bammetrics/src/main/scala/nl/lumc/sasc/biopet/pipelines/bammetrics/BammetricsReport.scala

```diff
@@ -42,13 +41,15 @@ object BammetricsReport extends ReportBuilder {
   /** Name of report */
   val reportName = "Bam Metrics"
 
+  def pipelineName = "bammetrics"
+
   /** Root page for single BamMetrcis report */
-  def indexPage = {
-    val bamMetricsPage = this.bamMetricsPage(summary, sampleId, libId)
+  def indexPage: Future[ReportPage] = Future {
```

Comment:
What happens if we do `this.bamMetricsPage(...).map` here, instead of creating a new `Future { ... }` block that has `Await` in it?